projects/ion/src/lib/divider/divider.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+/* tslint:disable:no-unused-variable */
+import { render, screen } from '@testing-library/angular';
+import { DividerComponent, IonDividerProps } from './divider.component';
+
+const ClassType = {
+  solid: 'ion-divider-solid',
+  dashed: 'ion-divider-dashed',
+  text: 'ion-divider-text',
+  vertical: 'ion-divider-vertical',
+  horizontal: 'ion-divider-horizontal',
+};
+
+const sut = async (customProps?: IonDividerProps) => {
+  await render(DividerComponent, {
+    componentProperties: customProps,
+  });
+  return screen.findByTestId('hr');
+};
+
+describe('DividerComponent', () => {
+  it('should render divider with default', async () => {
+    const divider = await sut({});
+    expect(divider).toHaveClass(ClassType['solid']);
```

Comment:
```suggestion
    expect(divider).toHaveClass(ClassType.solid);
```
Just a suggestion, I think it's better to read this way. What do you think?